services-validation.md

```diff
@@ -309,7 +309,7 @@ Passing `NULL` as a "where" clause value will add a check for a `NULL` database
 <a name="rule-image"></a>
 #### image
 
-The file under validation must be an image (jpeg, png, bmp, or gif)
+The file under validation must be an image (jpeg, png, bmp, svg or gif)
```

Comment:
Is this correct? I thought October still doesn't allow SVGs unless explicitly enabled in the config.

Reply:
The validation itself comes from laravel and svg are allowed